build.gradle

```diff
@@ -48,4 +48,10 @@ task generateMigration {
 		def migrationId = DateTimeFormatter.ofPattern('yyyyMMddkkmmssSSS').format(ZonedDateTime.now(ZoneOffset.UTC))
 		new File("src/main/resources/db/migration/V${migrationId}__${mName}.sql").write('')
 	}
+}
+
+flyway {
+	url = 'jdbc:postgresql://localhost:5432/T9AgeCombat'
```

Comment:
this works for me too if 'postgres' is too generic

Reply:
I  can swap it back to what you had in your PR.